corehq/messaging/smsbackends/amazon_pinpoint/models.py

```diff
@@ -0,0 +1,64 @@
+import boto3
+from corehq.apps.sms.models import SQLSMSBackend
+from corehq.apps.sms.util import clean_phone_number
+from corehq.messaging.smsbackends.amazon_pinpoint.forms import PinpointBackendForm
+
+MESSAGE_TYPE = "TRANSACTIONAL"
+
+
+class SQLPinpointBackend(SQLSMSBackend):
```

Comment:
SQL prefix not necessary
```suggestion
class PinpointBackend(SQLSMSBackend):
```

Reply:
Thanks. Fixed for Pinpoint and Infobip backends.